src/theme/styles.ts

```diff
@@ -46,6 +54,32 @@ const styles = {
       paddingInlineStart: 5,
     },
     "@keyframes fadeIn": { "0%": { opacity: 0 }, "100%": { opacity: 1 } },
+
+    ".notion-page": {
+      width: "var(--chakra-sizes-full) !important",
+      "@media (min-width : 1080px)": {
+        borderRadius: "15px",
+        background: "rgba(63,63,70, 0.45)",
```

Comment:
As I said before (in another comment), I think we could use `var(--chakra-colors-whiteAlpha-50)` here